src/tests/ftest/scripts/main.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
Isn't this still needed?

Reply:
No.  This file is not executed.  It is included in a remote command, effetively with a `cat` (although it is a `sed` not a `cat` but the net result is the same): https://github.com/daos-stack/daos/blob/ecca8a53c23cec4f59da930701c91dede2b804b0/ftest.sh#L157